style.css

```
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: large;
+}
+
+#main {
+  background-color: #f5f5f5;
+  width: 100%;
+  text-align: center;
+  padding: 1.5rem;
```

I think instead of using rem we can use px for padding.
```suggestion
  padding: 20px;
```

---

- [ ] As I mentioned in my previous comments, Please provide in your comment to the student the reason why you asking to use "article " :smiley:

---

I think instead of using rem we can use px for padding , as rem is mainly used in newer browser and px in  older browser. 

If you want to use rem, should first declare px before using rem(Root EM) as 1rem equals font size of html element (by default which is 16px). 😃